File: examples/src/Charts/GetWidthHeightFromChartPlotArea.cs
using System.IO

import aspose.slides as slides
using Aspose.slides.Charts
import aspose.pydrawing as drawing
using Aspose.slides.Export

namespace Aspose.slides.Examples.CSharp.Charts
{
    public class GetWidthHeightFromChartPlotArea
    {
        public static void Run()
        {
            #ExStart:GetWidthHeightFromChartPlotArea
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Charts()

            using (Presentation pres = new Presentation(dataDir+"test.Pptx"))
            {
                Chart chart = (Chart)pres.slides[0].shapes.add_chart(slides.charts.ChartType.CLUSTERED_COLUMN, 100, 100, 500, 350)
                chart.validate_chart_layout()

                double x = chart.plot_area.ActualX
                double y = chart.plot_area.ActualY
                double w = chart.plot_area.ActualWidth
                double h = chart.plot_area.ActualHeight

                # Save presentation with chart
                pres.save(dataDir + "Chart_out.pptx", slides.export.SaveFormat.PPTX)
            }
                        
            
            #ExEnd:GetWidthHeightFromChartPlotArea
        }
    }
}

File: examples/src/Charts/GetWidthHeightFromChartPlotArea.py
import aspose.slides as slides
import aspose.pydrawing as drawing

def charts_get_width_height_from_chart_plot_area():
    #ExStart:GetWidthHeightFromChartPlotArea
    # The path to the documents directory.
    outDir = "./examples/out/"

    with slides.Presentation() as pres:
        chart = pres.slides[0].shapes.add_chart(slides.charts.ChartType.CLUSTERED_COLUMN, 100, 100, 500, 350)
        chart.validate_chart_layout()

        x = chart.plot_area.actual_x
        y = chart.plot_area.actual_y
        w = chart.plot_area.actual_width
        h = chart.plot_area.actual_height

        # Save presentation with chart
        pres.save(outDir + "charts_get_width_height_from_chart_plot_area_out.pptx", slides.export.SaveFormat.PPTX)

    #ExEnd:GetWidthHeightFromChartPlotArea